app/uk/gov/hmrc/transitmovementsrouter/connectors/EISConnector.scala

```diff
@@ -91,11 +91,7 @@ class EISConnectorImpl(
     s"${HTTP_DATE_FORMATTER.format(OffsetDateTime.now(clock.withZone(ZoneOffset.UTC)))} UTC"
 
   def shouldCauseCircuitBreakerStrike(result: Try[Either[RoutingError, Unit]]): Boolean =
-    result match {
-      case Success(Left(DuplicateLRNError(_, _, _))) => false // Not to open circuit breaker for DuplicateLRN error
-      case Success(_)                                => true
-      case _                                         => false
-    }
+    result.map(_.isLeft).getOrElse(true)
```

Comment:
We don't want a circuit breaker strike for a duplicate LRN error -- why go back to this?

Reply:
Yes @drnaylor-oc this is controlled in retryingOnFailures method & circuit breaker won't be triggered.

Reply:
@drnaylor-oc Thank you for the detailed explanation. I understood it bit slightly different as when I tested it didn't do the retry & didn't go inside the circuit breaker and now it makes more clear now. Updated the code and will be pushing the changes after tests run.